contracts/masset/BasketManager.sol

```diff
@@ -485,6 +503,68 @@ contract BasketManager is
         emit BassetRemoved(bAsset.addr);
     }
 
+    /**
+     * @dev Transfers all collateral from one lending market to another - used initially
+     *      to handle the migration between Aave V1 and Aave V2. Note - only supports non
+     *      tx fee enabled assets
+     * @param _bAssets Array of basket assets to migrate
+     * @param _newIntegration Address of the new platform integration
+     */
+    function migrateBassets(
+        address[] calldata _bAssets,
+        address _newIntegration
+    )
+        external
+        onlyGovernor
+    {
+        uint256 len = _bAssets.length;
+        require(len > 0, "Must migrate some bAssets");
+
+        for(uint i = 0; i < len; i++){
+            // 1. Check that the bAsset is in the basket
+            address bAsset = _bAssets[i];
+            (bool inBasket, uint8 index) = _isAssetInBasket(bAsset);
+            require(inBasket, "bAsset does not exist");
+
```

Comment:
Maybe it would make sense to add something like `require(!basket.bassets[index].isTransferFeeCharged` here.